cremalab/sarah-todo
# Adasp 16 layout

## Description

basic layout of todo page

### 🔗 Reference

> _Relevant links (e.g. issue, design, etc.)_

- [Jira Board ADASP-16-layout](https://crema.atlassian.net/jira/software/projects/ADASP/boards/108?selectedIssue=ADASP-16)

### 📋 Acceptance Criteria

> _Checked off by PR **Assignees**_

- [x] to-do list is visible
- [x] total count appears
- [x] header says "My tasks"
- [x] new task form has an input and Save/Cancel buttons
- [x] created tasks have edit/delete buttons





### 👩‍🔬 Test Instructions

> _Provided by PR **Assignees**_

1. Click Create New Task button, count should increase from 0 to 1
2. Verify acceptance criteria

## 🔎 Reviewer Checklist

> _Checked off by PR **Reviewers**_




---

> ### Roles & Responsibilities
>
> #### 👨‍💻 Assignee
>
> - Initiator of this PR (be sure to set in GitHub UI)
> - Addresses feedback and change requests
> - Merges PR once approved (usually deletes branch unless `develop` or `release`)
>
> #### 👩‍💻 Reviewer
>
> - Invited to review PR by **Assignee** (via GitHub UI)
> - Is expected to complete a review and address followup
